wls-eai-service/src/main/java/de/muenchen/oss/wahllokalsystem/eaiservice/domain/wahlbeteiligung/WahlbeteiligungRepository.java

```diff
@@ -0,0 +1,10 @@
+package de.muenchen.oss.wahllokalsystem.eaiservice.domain.wahlbeteiligung;
+
+import java.util.UUID;
+import org.springframework.data.repository.CrudRepository;
+
+public interface WahlbeteiligungRepository extends CrudRepository<Wahlbeteiligung, UUID> {
+
+    @Override
+    <S extends Wahlbeteiligung> S save(S wahlbeteiligung);
```

Comment:
Die Definition ist nicht notwendig da sie bereits aus dem CrudRepo kommt. Bei anderen Service haben wir es um die notwendigen Rechte zu definieren.

```suggestion
public interface WahlbeteiligungRepository extends CrudRepository<Wahlbeteiligung, UUID> {
```